Traceback (most recent call last):
  File "C:\Users\miaha\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\miaha\anaconda3\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\miaha\anaconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\miaha\anaconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\miaha\anaconda3\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:\Users\miaha\anaconda3\lib\site-packages\nbclient\client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "C:\Users\miaha\anaconda3\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\miaha\anaconda3\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
corpus_root = os.getcwd() + '/data/characters/'
file_list = characters_dropna['File_Name'] + '.txt'
corpus = pcr(corpus_root, file_list)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_8332/1500867459.py[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [0mcorpus_root[0m [1;33m=[0m [0mos[0m[1;33m.[0m[0mgetcwd[0m[1;33m([0m[1;33m)[0m [1;33m+[0m [1;34m'/data/characters/'[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 2[1;33m [0mfile_list[0m [1;33m=[0m [0mcharacters_dropna[0m[1;33m[[0m[1;34m'File_Name'[0m[1;33m][0m [1;33m+[0m [1;34m'.txt'[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      3[0m [0mcorpus[0m [1;33m=[0m [0mpcr[0m[1;33m([0m[0mcorpus_root[0m[1;33m,[0m [0mfile_list[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;31mNameError[0m: name 'characters_dropna' is not defined
NameError: name 'characters_dropna' is not defined

